src/Tools/ExternalAccess/FSharp/Completion/CompletionOptions.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.FSharp.Completion
```

Comment:
>.Completion [](start = 54, length = 11)

Do we want the subnamespaces of `Microsoft.CodeAnalysis.ExternalAccess.FSharp`? Seems unnecessary.

Reply:
➡️ This is completely at the discretion of the F# team.

Reply:
I'm fine with it being like this.